tasks/system_probe.py

```diff
@@ -33,6 +33,11 @@
 KITCHEN_ARTIFACT_DIR = os.path.join(KITCHEN_DIR, "site-cookbooks", "dd-system-probe-check", "files", "default", "tests")
 TEST_PACKAGES_LIST = ["./pkg/ebpf/...", "./pkg/network/...", "./pkg/collector/corechecks/ebpf/..."]
 TEST_PACKAGES = " ".join(TEST_PACKAGES_LIST)
+TEST_TIMEOUTS = {
+    "pkg/network/tracer$": "0",
+    "pkg/network/protocols/http$": "0",
+    "pkg/network/protocols": "5m",
```

Comment:
Is 5 minutes sufficient for protocol classification?

Reply:
This is for each package, and what CI already uses.